ledger-api/grpc-definitions/com/daml/ledger/api/v2/reassignment.proto

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v2;
+
+import "com/daml/ledger/api/v1/event.proto";
+import "google/protobuf/timestamp.proto";
+
+option java_outer_classname = "ReassignmentOuterClass";
+option java_package = "com.daml.ledger.api.v2";
+option csharp_namespace = "Com.Daml.Ledger.Api.V2";
+
+// Complete view of an on-ledger reassignment.
+message Reassignment {
```

Comment:
Should this one be ReassignmentEvent ?

Reply:
I think the current naming is aligned to Transaction/CreatedEvent (so stream entry container / event).